server/model/time.js

```diff
@@ -0,0 +1,4 @@
+export type Time = {|
+  text: string;
+  timestamp: string;
```

Comment:
Did try that, but ran into issues - what it actually is, is a string with a specific format to be used in the <time> datetime attribute. Can we discuss this when you have a minute?